soc/arm/atmel_sam0/samd20/Kconfig.series

```diff
@@ -5,11 +5,11 @@
 
 config SOC_SERIES_SAMD20
 	bool "Atmel SAMD20 MCU"
-	select CPU_CORTEX_M
```

Comment:
Any reason for this removal?

Reply:
I think this was redundant, since CPU_CORTEX_M0PLUS selects CPU_CORTEX_M